cmake/AppImageUpdateConfig.cmake.in

```diff
@@ -1,8 +1,19 @@
 @PACKAGE_INIT@
 
 # look up dependencies
-find_package(zsync2 REQUIRED)
-find_package(libappimage REQUIRED)
-find_package(Qt5 REQUIRED COMPONENTS Core Widgets)
+set(APPIMAGE_UPDATE_USE_SYSTEM_ZSYNC2 @USE_SYSTEM_ZSYNC2@)
```

Comment:
I think we don't even need those temporary variables. Also, as said before, I'm missing some `unset` calls here, and I'd give the variables a prefix.

I think using boolean generator expressions, though, we don't need any of these variables:

```cmake
if($<BOOL:${USE_SYSTEM_ZSYNC2}>)
```
The expression will be converted to an integer (0 or 1, depending on the value). The advantage of this solution is that the strings in `USE_SYSTEM_ZSYNC2` will not be propagated into a file that is installed later.

Reply:
The variables are not meant to be temporary. Instead a would like to inform libappimageupdate clients how it was built.

Reply:
These should be documented in the `README` then, I guess, and maybe also in this file.

Please use the generator expressions to give the variables uniform values.

Also, please remove the underscore between AppImage and Update. The tool is called "AppImageUpdate" without a space.